packages/notifi-dapp-example/src/components/TopicOptions.tsx

```diff
@@ -213,31 +254,57 @@ export const TopicOptions = <T extends TopicRowCategory>(
                   }
                   selectOrInputValue(evt.target.value);
                 }
+                if (evt.target.value === '') {
+                  return options.includes(selectedOption?.toString() ?? '')
+                    ? setCustomInput('')
+                    : setCustomInput(selectedOption?.toString() ?? '');
+                }
+                // Case#2: If input === existing selected option, set customInput to empty
+                if (
+                  options.includes(evt.target.value) &&
+                  selectedOption === evt.target.value
+                )
+                  return setCustomInput('');
+
+                // Case#3: Subscribe new value
+                const { value } = validateCustomInputRange(
+                  evt.target.value,
+                  props.userInputParam.customInputConstraints,
+                );
+                setCustomInput(value.toString());
+                selectOrInputValue(value);
               }}
-              placeholder={props.placeholder ?? 'Custom'}
+              placeholder={
+                props.userInputParam.kind === 'price'
+                  ? 'Enter Price'
```

Comment:
Hmmm .... I guess we will no longer have price kind. Shall we make a more natural term to fallback to?

Reply:
We'll still have price kind for trading pair alert